modelling/src/test/java/org/axonframework/modelling/command/AnnotationCommandTargetResolverTest.java

```diff
@@ -16,159 +16,282 @@
-            }
-        }));
-    }
-
-    @Test
-    public void testResolveTarget_WithAnnotatedFields() {
-        final UUID aggregateIdentifier = UUID.randomUUID();
-        final Object version = 1L;
-        VersionedAggregateIdentifier actual = testSubject.resolveTarget(
-                asCommandMessage(new FieldAnnotatedCommand(aggregateIdentifier, version)));
-        assertEquals(aggregateIdentifier.toString(), actual.getIdentifier());
-        assertEquals(version, actual.getVersion());
-    }
-
-    @Test
-    public void testResolveTarget_WithAnnotatedFields_StringIdentifier() {
-        final UUID aggregateIdentifier = UUID.randomUUID();
-        final Object version = 1L;
-        VersionedAggregateIdentifier actual = testSubject.resolveTarget(
-                asCommandMessage(new FieldAnnotatedCommand(aggregateIdentifier, version)));
-        assertEquals(aggregateIdentifier.toString(), actual.getIdentifier());
-        assertEquals(version, actual.getVersion());
-    }
-
-    @Test
-    public void testResolveTarget_WithAnnotatedFields_ObjectIdentifier() {
-        final Object aggregateIdentifier = new Object();
-        final Object version = 1L;
-        VersionedAggregateIdentifier actual = testSubject.resolveTarget(
-                asCommandMessage(new FieldAnnotatedCommand(aggregateIdentifier, version)));
-        assertEquals(aggregateIdentifier.toString(), actual.getIdentifier());
-        assertEquals(version, actual.getVersion());
-    }
-
-    @Test
-    public void testResolveTarget_WithAnnotatedFields_ParsableVersion() {
-        final UUID aggregateIdentifier = UUID.randomUUID();
-        final Object version = "1";
-        VersionedAggregateIdentifier actual = testSubject.resolveTarget(
-                asCommandMessage(new FieldAnnotatedCommand(aggregateIdentifier, version)));
-        assertEquals(aggregateIdentifier.toString(), actual.getIdentifier());
-        assertEquals((Long) 1L, actual.getVersion());
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void testResolveTarget_WithAnnotatedFields_NonNumericVersion() {
-        final UUID aggregateIdentifier = UUID.randomUUID();
-        final Object version = "abc";
-        testSubject.resolveTarget(asCommandMessage(new FieldAnnotatedCommand(aggregateIdentifier, version)));
-    }
-
-    private static class FieldAnnotatedCommand {
-
-        @TargetAggregateIdentifier
-        private final Object aggregateIdentifier;
-
-        @TargetAggregateVersion
-        private final Object version;
-
-        public FieldAnnotatedCommand(Object aggregateIdentifier, Object version) {
-            this.aggregateIdentifier = aggregateIdentifier;
-            this.version = version;
-        }
-
-        public Object getAggregateIdentifier() {
-            return aggregateIdentifier;
-        }
-
-        public Object getVersion() {
-            return version;
-        }
-    }
-}
+	private AnnotationCommandTargetResolver testSubject;
```

Comment:
I believe you've incorrectly reindented the file. Would you mind reverting/fixing that?
It makes it very hard to see which tests you've added.